django/text_extractor/models.py

```diff
@@ -18,6 +18,7 @@ class OutputFile(SecureModel):
     user_request = models.ForeignKey(
         UserRequest, related_name="output_files", on_delete=models.CASCADE
     )
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Previously it was using UserRequest model's creation time which messed up the times. Added creation time to OutputFile model now.

Reply:
Isn't it OK to use UserRequest creation time? They should only differ by at most like 1 minute right?
See my comment above about, "why not delete UserRequest".

Reply:
Deleting userRequest works fine, but in the test it is not automatically deleting the files as it is doing normally with the django cleanup. Any suggestions on how to use the django cleanup on tests?